app/src/cbor.h

```diff
@@ -153,4 +153,39 @@ namespace scitt::cbor
     output.shrink_to_fit();
     return output;
   }
+
+  // see https://www.ietf.org/rfc/rfc9679.html
+  inline std::vector<uint8_t> cose_key_to_cbor(
```

Comment:
```suggestion
  inline std::vector<uint8_t> ec_cose_key_to_cbor(
```
other kty have different params.

Reply:
separated into ec_cose_key_to_cbor and rsa_cose_key_to_cbor and added  basic tests